html/semantics/scripting-1/the-script-element/data-url.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>data URL and scripts</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id=log></div>
+<script>
+  setup({allow_uncaught_exception:true})
+  var t = async_test()
+  t.step(() => {
```

Comment:
`async_test(t => { ... })` saves a line and one need not look to see if `t` is referenced elsewhere.